TelCo.ColorCoder/Pair.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Drawing;
+
+namespace TelCo.ColorCoder
+{ 
+    class Pair
+    {
+        internal Color majorColor;

```

Comment:
ColorPair was already given in the exercise, 
it is not a good practice to expose memper variables directly.
but if a class is just a place holder for variables like this class, its good to include default property accessors
internal Color MajorColor {get; set;}

A suggestion to improve the code that was given.. you need not implement